### ISSUE TITLE
meshroom don't calculate prepare dense scene

### ISSUE THREAD

**Issue by AlejandroHerreraJ** [alicevision/meshroom#456]
I´m trying to make a project on Meshroom, but when meshroom were calculating  it stops on PrepareDenseScene

[6/11](1/2) PrepareDenseScene
HDF5-DIAG: Error detected in HDF5 (1.10.1) thread 0:
  #000: C:\GitHub\vcpkg\buildtrees\hdf5\src\CMake-hdf5-1.10.1\hdf5-1.10.1\src\H5F.c line 408 in H5Fis_hdf5(): unable open file
    major: File accessibilty
    minor: Not an HDF5 file
  #001: C:\GitHub\vcpkg\buildtrees\hdf5\src\CMake-hdf5-1.10.1\hdf5-1.10.1\src\H5Fint.c line 532 in H5F__is_hdf5(): unable to open file
    major: Low-level I/O
    minor: Unable to initialize object
  #002: C:\GitHub\vcpkg\buildtrees\hdf5\src\CMake-hdf5-1.10.1\hdf5-1.10.1\src\H5FD.c line 809 in H5FD_open(): open failed
    major: Virtual File Layer
    minor: Unable to initialize object
  #003: C:\GitHub\vcpkg\buildtrees\hdf5\src\CMake-hdf5-1.10.1\hdf5-1.10.1\src\H5FDsec2.c line 346 in H5FD_sec2_open(): unable to open file: name = 'C:/Users/aleja/OneDrive - Universidad de los andes/Escritorio/Universidad/Dise├▒o Grafico/Inventor/Proyecto 2/MeshroomCache/StructureFromMotion/3b02f27ac03ad0842d9412800084ccab34c7569c/sfm.abc', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0
    major: File accessibilty
    minor: Unable to open file
 - commandLine: aliceVision_prepareDenseScene  --input "C:/Users/aleja/OneDrive - Universidad de los andes/Escritorio/Universidad/Diseño Grafico/Inventor/Proyecto 2/MeshroomCache/StructureFromMotion/3b02f27ac03ad0842d9412800084ccab34c7569c/sfm.abc" --outputFileType exr --saveMetadata True --saveMatricesTxtFiles False --verboseLevel info --output "C:/Users/aleja/OneDrive - Universidad de los andes/Escritorio/Universidad/Diseño Grafico/Inventor/Proyecto 2/MeshroomCache/PrepareDenseScene/c4c01d67d6f577b80b31307dbd1a3a773abd4a0d" --rangeStart 0 --rangeSize 40
 - logFile: C:/Users/aleja/OneDrive - Universidad de los andes/Escritorio/Universidad/Diseño Grafico/Inventor/Proyecto 2/MeshroomCache/PrepareDenseScene/c4c01d67d6f577b80b31307dbd1a3a773abd4a0d/0.log
 - elapsed time: 0:00:07.485765
ERROR:root:Error on node computation: Error on node "PrepareDenseScene_1(0)":
Log:
Program called with the following parameters:
 * input = "C:/Users/aleja/OneDrive - Universidad de los andes/Escritorio/Universidad/Diseño Grafico/Inventor/Proyecto 2/MeshroomCache/StructureFromMotion/3b02f27ac03ad0842d9412800084ccab34c7569c/sfm.abc"
 * output = "C:/Users/aleja/OneDrive - Universidad de los andes/Escritorio/Universidad/Diseño Grafico/Inventor/Proyecto 2/MeshroomCache/PrepareDenseScene/c4c01d67d6f577b80b31307dbd1a3a773abd4a0d"
 * outputFileType = "exr"
 * rangeSize = 40
 * rangeStart = 0
 * saveMatricesTxtFiles = 0
 * saveMetadata = 1
 * verboseLevel = "info"

**Comment by natowi** [comment]
The problem _might_ be with your project/image path: Dise├▒o Grafico
Try again with a file path without special characters and/or with a local (not OneDrive) folder.

**Comment by alvaroperdigao** [comment]
> The problem might be with your project/image path: Dise├▒o Grafico
> Try again with a file path without special characters and/or with a local (not OneDrive) folder.
> #201

This works for me! Thank you so much!

I put the files in a folder in the root drive.

**Comment by LucasLiSan** [comment]
> The problem _might_ be with your project/image path: Dise├▒o Grafico
> Try again with a file path without special characters and/or with a local (not OneDrive) folder.
> #209

Funcionou, muito obrigado.
worked, thank you.

**Comment by csiszibeni** [comment]
Hy everyone, 

my problem during the "Prepare Dense Scene":
Status:
{
    "status": "ERROR",
    "execMode": "LOCAL",
    "nodeName": "PrepareDenseScene_1",
    "nodeType": "PrepareDenseScene",
    "packageName": "aliceVision",
    "packageVersion": null,
    "graph": "",
    "commandLine": "aliceVision_prepareDenseScene  --input \"C:/Users/Csisz\u00e1r Benj\u00e1min/Desktop/Kutya/MeshroomCache/
StructureFromMotion/1fbd28971f53e7cf97406c3d0f9bcfc3fdc76e49/sfm.abc\" --outputFileType exr --saveMetadata True --
saveMatricesTxtFiles False --evCorrection False --verboseLevel info --output \
"C:/Users/Csisz\u00e1r Benj\u00e1min/Desktop/Kutya/MeshroomCache/PrepareDenseScene/052e305120522b6913a1af1d3f9a99a0d078bc78\" --
rangeStart 0 --rangeSize 40",
    "env": null,
    "startDateTime": "2020-02-20 09:08:42.114785",
    "endDateTime": "",
    "elapsedTime": 0,
    "hostname": "DESKTOP-RNCL7BV",
    "sessionUid": "9176c0c2-53b7-11ea-9929-7085c2fd08d3",
    "returnCode": 3221226505,
    "elapsedTimeStr": "0:00:00"
}

Output:
Program called with the following parameters:
 * evCorrection = 0
 * input = "C:/Users/Csisz�r Benj�min/Desktop/Kutya/MeshroomCache/StructureFromMotion/1fbd28971f53e7cf97406c3d0f9bcfc3fdc76e49/sfm.abc"
 * output = "C:/Users/Csisz�r Benj�min/Desktop/Kutya/MeshroomCache/PrepareDenseScene/052e305120522b6913a1af1d3f9a99a0d078bc78"
 * outputFileType = "exr"
 * rangeSize = 40
 * rangeStart = 0
 * saveMatricesTxtFiles = 0
 * saveMetadata = 1
 * verboseLevel = "info"

Exporting Scene Undistorted Images
0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
[09:08:42.294825][info] median Ev : 4.75889

What can I do? 

Thanks in advice,
Beni

**Comment by fabiencastan** [comment]
PrepareDenseScene does not support paths to your images and project file with non-ascii characters, as you have in your username.

**Comment by csiszibeni** [comment]
Thank you for your answer. 
I am not a big computer master. Can you help me, what sholuld I do now? What sholuld I change.

**Comment by fabiencastan** [comment]
Change your images files and projet file to a folder like "C:\photogrammetry\mydataset01" without any special character.

**Comment by csiszibeni** [comment]
Yeah, it is working! Thank you very much ;)

**Comment by JMolina3Dart** [comment]
Hello, it is the first time that I use this program and the truth is I do not know why I get this error, if someone can help, I would appreciate it ...

- commandLine: aliceVision_prepareDenseScene  --input "C:/Users/Usuario/Desktop/Blender/ImagenesFotogrametria/MeshroomCache/StructureFromMotion/7e7ce1314ff9c59540156ffc440b656f624c4ee3/sfm.abc" --outputFileType exr --saveMetadata True --saveMatricesTxtFiles False --evCorrection False --verboseLevel info --output "C:/Users/Usuario/Desktop/Blender/ImagenesFotogrametria/MeshroomCache/PrepareDenseScene/7b827bf4783a79b3efdf34fb3f6bcfe6cc3ba447" --rangeStart 0 --rangeSize 40
 - logFile: C:/Users/Usuario/Desktop/Blender/ImagenesFotogrametria/MeshroomCache/PrepareDenseScene/7b827bf4783a79b3efdf34fb3f6bcfe6cc3ba447/0.log
ERROR:root:Error on node computation: Error on node "PrepareDenseScene_1(0)":
Log:
Program called with the following parameters:
 * evCorrection = 0
 * input = "C:/Users/Usuario/Desktop/Blender/ImagenesFotogrametria/MeshroomCache/StructureFromMotion/7e7ce1314ff9c59540156ffc440b656f624c4ee3/sfm.abc"
 * output = "C:/Users/Usuario/Desktop/Blender/ImagenesFotogrametria/MeshroomCache/PrepareDenseScene/7b827bf4783a79b3efdf34fb3f6bcfe6cc3ba447"
 * outputFileType = "exr"
 * rangeSize = 40
 * rangeStart = 0
 * saveMatricesTxtFiles = 0
 * saveMetadata = 1
 * verboseLevel = "info"

Exporting Scene Undistorted Images
0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
[23:16:43.196125][info] Median Camera Exposure: 0.79996, Median EV: 0.322

WARNING:root:Downgrade status on node "DepthMap_1(0)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(1)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(2)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(3)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(4)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(5)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(6)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(7)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(8)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(9)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(10)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(11)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMap_1(12)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "Texturing_1" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMapFilter_1(0)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMapFilter_1(1)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMapFilter_1(2)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "DepthMapFilter_1(3)" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "Meshing_1" from Status.SUBMITTED to Status.NONE
WARNING:root:Downgrade status on node "MeshFiltering_1" from Status.SUBMITTED to Status.NONE

**Comment by V-Tornados** [comment]
Hello I'm having the same problem here. Except, I'm not certain if I'm doing the right thing. I highlighted it in red if that's the image file you guys are referring to? I typed in what fabiencastan posted. But, it doesn't work. I'm not sure what I'm doing wrong. So I conceded on doing it on my own, and request help from this forum If you guys can send me an image or vdeo on what I exactly need to do. That'll will be great.

P.S. I'm not a computer genesis don't know what alot of the terms. But can only decipher what they mean.
![Meshroom](https://user-images.githubusercontent.com/88093599/127333138-ddac6c7a-4281-4eda-ba1d-2bf063976335.jpg)
